nautobot/extras/forms.py

```diff
@@ -1048,7 +1048,8 @@ class JobResultFilterForm(BootstrapMixin, forms.Form):
         queryset=Job.objects.all(),
         required=False,
         to_field_name="slug",
-        widget=APISelectMultiple(api_url="/api/extras/job-models/"),
+        widget=APISelectMultiple(api_url="/api/extras/jobs/", api_version="1.3"),
+        brief_mode=False,
```

Comment:
Why do we need `brief_mode=False` here?

Reply:
Slug does not show up if `brief_mode=True`

Reply:
Thought that might be the issue... I think that's a bug in the implementation of `NestedJobSerializer`, as it looks like pretty much all of our other `Nested*Serializer` classes *do* include `slug`. Let's fix the serializer so that `fields` includes `"slug"` and then brief_mode should work as expected.